specification/vmware/Microsoft.AVS.Management/tspconfig.yaml

```diff
@@ -18,41 +18,41 @@ options:
     omit-unreachable-types: true
     use-read-only-status-schema: true
 
-  # "@azure-tools/typespec-csharp":
-  #   flavor: azure
-  #   package-dir: "Azure.ResourceManager.Avs"
-  #   clear-output-folder: true
-  #   model-namespace: true
-  #   namespace: "{package-dir}"
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    package-dir: "Azure.ResourceManager.Avs"
+    clear-output-folder: true
+    model-namespace: true
+    namespace: "{package-dir}"
 
-  # "@azure-tools/typespec-go":
-  #   service-dir: "sdk/resourcemanager/avs"
-  #   package-dir: "armavs"
-  #   module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
-  #   fix-const-stuttering: true
-  #   flavor: "azure"
-  #   generate-samples: true
-  #   generate-fakes: true
-  #   head-as-boolean: true
-  #   inject-spans: true
+  "@azure-tools/typespec-go":
+    service-dir: "sdk/resourcemanager/avs"
+    package-dir: "armavs"
+    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    fix-const-stuttering: true
+    flavor: "azure"
+    generate-samples: true
+    generate-fakes: true
+    head-as-boolean: true
+    inject-spans: true
 
-  # "@azure-tools/typespec-java":
-  #   package-dir: "azure-resourcemanager-avs"
-  #   namespace: "com.azure.resourcemanager.avs"
-  #   service-name: "avs"
-  #   flavor: azure
+  "@azure-tools/typespec-java":
+    package-dir: "azure-resourcemanager-avs"
+    namespace: "com.azure.resourcemanager.avs"
+    service-name: "avs"
+    flavor: azure
 
-  # "@azure-tools/typespec-python":
-  #   package-dir: "azure-mgmt-avs"
-  #   namespace: "azure.mgmt.avs"
-  #   flavor: "azure"
-  #   generate-test: true
-  #   generate-sample: true
+  "@azure-tools/typespec-python":
+    package-dir: "azure-mgmt-avs"
```

Comment:
```suggestion
    service-dir: "sdk/compute"
    package-dir: "azure-mgmt-avs"
```

Reply:
@ChenxiJiang333 avs is already in `sdk/compute` folder: https://github.com/Azure/azure-sdk-for-python/tree/b484b70cd7d4122cb74a7a749c551cb21add70ad/sdk/compute/azure-mgmt-avs